src/printapp/templates/index.html

```diff
@@ -4,69 +4,64 @@
   
   <script type="text/x-handlebars" data-template-name="application">
   {{#if isAuthenticated}}
-    {{partial "welcome"}}
+	{{partial "welcome"}}
     {{render "cloudprint" cloudprint}}
     {{partial "queue"}}
-    {{partial "budget"}}
   {{else}}
     {{render "signIn" signIn}}
   {{/if}}
-  {{render "printers" printers}}
-  {{partial "prices"}}
-  {{outlet}}
+    {{render "printers" printers}}
+  {{#if isAuthenticated}}
+    {{partial "budget"}}
+  {{/if}}
+    {{partial "prices"}}
+    {{outlet}}
   </script>
-  
-  <script type="text/x-handlebars" data-template-name="signIn">
-  <div class="row">
-    <div class="small-12 columns">
-      <h3 class="subheader">Welcome</h3>
-      <p>
-        Sign in with your Calvin username and passphrase.
-      </p>
-      <div class="row">
-        <div class="small-12 medium-8 columns">
-          <div id="signInError" data-alert hidden class="alert-box alert radius">
-            The username or passphrase you entered is incorrect.
-            <a href="#" class="close">&times;</a>
-          </div>
-          <div id="signInErrorUniflowDown" data-alert hidden class="alert-box alert radius">
-            <strong>Oops!</strong> We’re having trouble connecting to uniFLOW. It looks like uniFLOW may be down.
-            <a href="#" class="close">&times;</a>
-          </div>
-        </div>
-      </div>
-      <div class="row">
-        <div class="small-12 medium-4 columns">
-          <label>Username
-            {{input action='signin' type="text" value=inputUsername autofocus="autofocus" autocapitalize="off"}}
-          </label>
-          <label>Passphrase
-            {{input action='signin' type="password" value=inputPassword}}
-          </label>
-          <button {{action 'signin'}} {{bind-attr disabled=submitting}}>
-            {{#if submitting}}
-              Signing in...
-            {{else}}
-              Sign in
-            {{/if}}
-          </button>
-        </div>
-      </div>
-    </div>
-  </div>
 
-  <hr />
+  <script type="text/x-handlebars" data-template-name="signIn">
+	<div class="login-section">
+	  <div class="row">
+		<div class="small-12 columns">
+		  <h1 class="headline">Always get your papers in on time.</h1>
```

Comment:
@tylervz , what are your thoughts on this tag line? I generally prefer not to change marketing copy unless we have a good reason.